src/configurations/frontend-component-information.js

```diff
@@ -0,0 +1,27 @@
+export default {
```

Comment:
Is this information static? How should we update it? Should we document this?

Reply:
Most of the information will not change. I’m thinking about fetching it dynamically and will implement and document it.